index.html

```diff
@@ -29,9 +29,120 @@ <h1>Felipe<br>Oyarzo</h1>
                 </p>
             </div>
             <button class="button" type="button" id="btnHeadline">Start collaboration</button>
-            <div class="geometric"></div>            
+            <div class="geometric"></div>
         </section>
     </main>
+    <section class="projects">
+        <h1>Projects</h1>
+        <div class="grid-container">
+            <div class="card card1">
+                <img src="./assets/img/projects-img1.png" alt="Screenshot of project 1">
```

Comment:
- [ ] It is highly recommended to use background-image property in the css file for this image, not the `<img>` tag. Kindly fix that.